device/globals/scenes_cache.gd

```diff
@@ -1,4 +0,0 @@
-const scenes = [
```

Comment:
This file is still referenced in `global_vm.gd` on line 61:

https://github.com/mjtorn/escoria/blob/826c06f5a6e2170bde04f7b654cf0bebd037e868/device/globals/global_vm.gd#L61

Reply:
The declaration can remain, but the assignment should only happen if a scenes_cache has actually been set in the project settings. I think one should be set by default though, and I think it makes sense that the file is in `demo/`.